packages/react-ui-core/src/Form/RangeSlider.js

```diff
@@ -1,10 +1,17 @@
 import React, { Component } from 'react'
 import PropTypes from 'prop-types'
 import InputRange from 'react-input-range'
+import classnames from 'classnames'
+import themed from 'react-themed'
+
+@themed(/^RangeSlider/, {
```

Comment:
There's a `RangeSlider.css` I think that is going unused or needs to be updated?

Reply:
Yeah, I wasn't sure how to handle that file since it's really external.